### ISSUE TITLE
iOS: [a11y] Settings (gear icon) not reachable with VoiceOver

### ISSUE THREAD

**Issue by ypV8R8ifiFzTSX1E68w1jViw8TS6DesWDWTHKv7** [bitwarden/mobile#339]
The settings (gear) icon in the initial screen (login or create account) is not reachable nor spoken at all by VoiceOver, the screen reader for the blind on iOS.

I can swipe between three elements on that initial screen:
* The text "You have to login or regster to access your vault"
* The "login" button
* The "create account" button

Touching the left upper corner of the screen (where the gear icon resides) does not help, voiceover does not speak the settings icon at all, and also does not allow to invoke it.

To test this, you will likely have to enable VoiceOver in Accessibility menu of the iOS Settings application.

**Comment by closebot-bw** [comment]
This repo has been archived. To report an issue you're experiencing in the new native apps, visit the [iOS](https://github.com/bitwarden/ios) / [Android](https://github.com/bitwarden/android) repo.
